package main

import "exception/trydefer"

func main() {
	trydefer.DeferDemo()
}
